### ISSUE TITLE
DateTime AM/PM aria-label="&nbsp" - Can you give it a name like "AMPM"?

### ISSUE THREAD

**Issue by mayodele** [acidb/mobiscroll#246]
The DateTime AM/PM aria-label="&nbsp". Can you give it a name like "AMPM"? I am unable to find this element using jquery selection (Ex. $("[aria-label=' ']"))
# HTML for DateTime AMPM Wheel Below

![image](https://cloud.githubusercontent.com/assets/8651440/4145197/62a49754-33f3-11e4-9a04-0186b7ed9193.png)
